cmd/config/network-policy/nad-vlan.yml

```diff
@@ -0,0 +1,18 @@
+{{- $vlanId := add (div $.Iteration .namespaces_per_vlan) 1}}
```

Comment:
Is there a world where vlanId can be higher than 255 and then generates error in the JSON config?